pgbar.coffee

```diff
@@ -83,7 +83,9 @@ class PgbarItem
     if best_target != target
       target = best_target
       @target.html(formatNumber(target))
-      @needed.html(formatNumber(target - to_abs))
+
+    @needed.html(formatNumber(target - to_abs))
+
```

Comment:
I think this should actually grouped be with `@current` below and also be animated in `resetCounters` (counting down in this case).